client.ts

```diff
@@ -209,24 +223,25 @@ export class Collection<T> {
   async callApi(method: string, extra: Document): Promise<any> {
     const { endpoint, dataSource, headers } = this.client;
     const url = `${endpoint}/action/${method}`;
+    const body = this.client.stringify({
+      collection: this.name,
+      database: this.database.name,
+      dataSource: dataSource,
+      ...extra,
+    });
 
     const response = await this.client.fetch(url, {
       method: "POST",
       headers,
-      body: EJSON.stringify({
-        collection: this.name,
-        database: this.database.name,
-        dataSource: dataSource,
-        ...extra,
-      }),
+      body,
```

Comment:
this change was mostly for debugging and trying to write this statement in a single line, but fmt thinks that statement is still too long ... 80 char is so small...